src/Parser/GeoJsonParser.js

```diff
@@ -244,37 +214,51 @@ export default {
      * @typedef FeatureGeometry
      * @type {Object}
      *
-     * @property {string} type - Geometry type, can be <code>point</code>,
-     * <code>multipoint</code>, <code>linestring</code>,
-     * <code>multilinestring</code>, <code>polygon</code> or
-     * <code>multipolygon</code>.
-     * @property {Coordinates[]} vertices - All the vertices of the geometry.
-     * @property {?number[]} contour - If this geometry is a
-     * <code>polygon</code>, <code>contour</code> contains the indices that
-     * compose the contour (outer ring).
-     * @property {?Array} holes - If this geometry is a <code>polygon</code>,
-     * <code>holes</code> contains an array of indices representing holes in the
-     * polygon.
-     * @property {?Extent} extent - The 2D extent containing all the geometries.
-    */
+     * @property {Extent} extent - The 2D extent containing all the points
+     * composing the geometry.
+     * @property {?number[]} indices - If this geometry is a
+     * <code>polygon</code>, contains the indices that define the polygon. The
```

Comment:
I believe the indices are also defined for the linestrings (not just for polygons)

Reply:
Yes